File: src/main/java/me/kuku/yuq/service/QQLoginService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.QQLoginEntity;

import java.util.List;

@AutoBind
public interface QQLoginService {
    QQLoginEntity findByQQ(Long qq);
    void save(QQLoginEntity qqLoginEntity);
    void delByQQ(Long qq);
    List<QQLoginEntity> findByActivity();
}

File: src/main/kotlin/me/kuku/yuq/service/QQLoginService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.QQLoginEntity

@AutoBind
interface QQLoginService {
    fun findByQQ(qq: Long): QQLoginEntity?
    fun save(qqLoginEntity: QQLoginEntity)
    fun delByQQ(qq: Long)
    fun findByActivity(): List<QQLoginEntity>
}